### ISSUE TITLE
Missing swipe up/down buttons in TalkBack panel

### ISSUE THREAD

**Issue by koral--** [qbalsdon/android-ally-plugin#53]
To adjust Slider, ProgressBar, RatingBar etc. when TalkBack is enabled there are vertical swipe gestures. It may be tricky to perform them especially on emulator.

It will be useful if there are buttons for those actions in a TalkBack panel like Tap.

<img width="376" height="92" alt="Image" src="https://github.com/user-attachments/assets/3a6ff7ab-bcd3-44b3-8226-1bc0a09a9622" />

**Comment by qbalsdon** [comment]
Completed
